test/hotspot/jtreg/gc/g1/plab/TestPLABPromotion.java

```diff
@@ -72,7 +73,7 @@ public class TestPLABPromotion {
     private static final int PLAB_SIZE_HIGH = 65536;
     private static final int OBJECT_SIZE_SMALL = 10;
     private static final int OBJECT_SIZE_MEDIUM = 100;
-    private static final int OBJECT_SIZE_HIGH = 3258;
+    private static final int OBJECT_SIZE_HIGH = Platform.is64bit() ? 3266 : 3258;
```

Comment:
why this difference?

Reply:
I believe the value was carefully chosen to hit exactly a PLAB boundary. PLABs required some extra wiggle room so that we could fill them with the smallest possible object, which used to be 2 words. Now we can fill any remaining space, even if it's just 1 word, so we need to allocate 1 more word to exactly hit the PLAB boundary in the test.